fdbserver/workloads/DDBalance.actor.cpp

```diff
@@ -214,7 +214,6 @@ struct DDBalanceWorkload : TestWorkload {
 		state int currentBin = self->currentbin;
 		state int nextBin = 0;
 		state int i;
-		state int j;
```

Comment:
This is another case where it doesn't seem like `i` (at line 228/227) doesn't need to be state.

Reply:
Done